server/src/ClientToDBAPI/InvestigationInfo/mainRoute.ts

```diff
@@ -6,7 +6,7 @@ import { GET_INVESTIGATION_INFO } from '../../DBService/InvestigationInfo/Query'
 const investigationInfo = Router();
 
 investigationInfo.post('/staticInfo', (request: Request, response: Response) => {
-    graphqlRequest(GET_INVESTIGATION_INFO, { id: request.body.id })
```

Comment:
maybe it's better securitywise to use the first way?

Reply:
what way?

Reply:
{ id: request.body.id })